app/src/Components/Error/GraphQLError.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { UnauthorisedError } from './UnauthorisedError';
+
+class GraphQLError extends React.Component {
+  render() {
+    const error = this.props.error;
+    if (error.graphQLErrors.length > 0) {
+      return <div>
+        {error.graphQLErrors.statusCode + ': ' + error.graphQLErrors.result.message}
+      </div>;
+    }
+    if (error.networkError) {
+      switch (error.networkError.statusCode) {
+        case 401:
+          return <UnauthorisedError/>;
+        default:
+          return <div>
+            {error.networkError.statusCode + ': ' + error.networkError.result.message}
+          </div>;
+      }
+    }
+    return <div>No error. Something wrong happened...</div>;
```

Comment:
If something wrong happened, how can there be no error?

Reply:
Its an emergency return in case something else, apart from graphqlerror or networkError calls this module. 
I'd rather have an obscure message telling us something than a <div/> that would leave you with a black screen.
Once we design an error screen, most of these errors will probably be changed to a nice *svg.